docker-bench-security.sh

```diff
@@ -59,7 +59,7 @@ if [ -z "$logger" ]; then
 fi
 
 yell "# ------------------------------------------------------------------------------
-# Docker Bench for Security v1.0.0
+# Docker Bench for Security v1.1.0
```

Comment:
Good point. Let's leave it like this.

In the future, for actuary I think we should follow Docker versions instead /cc @zuBux